Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/CodeUtils.kt

```diff
@@ -54,7 +54,11 @@ fun String.quoteAsKotlinLiteral(): KotlinExpression {
 
     this.forEach { c ->
         val escaped = when (c) {
-            in 'a' .. 'z', in 'A' .. 'Z', in '0' .. '9', '_', '@', ':', '{', '}', '=', '[', ']', '^', '#', '*' -> c.toString()
+            in 'a' .. 'z', in 'A' .. 'Z',
+            in '0' .. '9',
+            '_', '@', ':', ';', '.', ',', '{', '}', '=', '[', ']', '^', '#', '*',
```

Comment:
What about adding all printables to set, as this loops seems to be pretty frequent and really slow now?

Reply:
It makes sense.